test/fixtures/asyncapi.yml

```diff
@@ -0,0 +1,42 @@
+# yaml-language-server: $schema=/Users/wookiee/sources/asyncapi-node/schemas/2.6.0-without-$id.json
```

Comment:
should we make this a more generic placeholder?

e.g. `/YOUR-HOME-DIRECTORY/sources/...`